worker/src/RTC/DataConsumer.cpp

```diff
@@ -93,24 +93,34 @@ namespace RTC
 	{
 		MS_TRACE();
 
-		flatbuffers::Offset<FBS::SctpParameters::SctpStreamParameters> sctpStreamParametersOffset;
```

Comment:
I've removed `Offset` suffix to be consistent. For instance, in `Producer::FillBuffer()` we don't add `Offset` suffix anywhere.

Reply:
@jmillan I think we should decide whether `Offset` suffix must be added or not. I see equivalent places where it's added and others in which it's not. IMHO it's redundant and unsightly and we should remove it from everywhere.

Reply:
Yes, I agree.